### PR TITLE
Feature: ShowAllListingsCommand

### PR DESCRIPTION
**Pull request by muhdbhz**
ShowAllListingsCommand require change to AddressBook related classes and ModelManager.

The above commit attempts to implement this command.

### CODE REVIEW

File: src/main/java/seedu/address/model/AddressBook.java [hunk 1]
```diff
@@ -75,6 +96,14 @@ public void addPerson(Person p) {
         persons.add(p);
     }
 
+    /**
+     * Adds a listing to the address book.
+     * The listing must not already exist in the address book.
+     */
+    public void addListing(Listing l) {
+        listings.add(l);
```
Comment (wang-h-z) [inline src/main/java/seedu/address/model/AddressBook.java:104]: Would be good to add the requreNonNull() check here as well.

**Review by wang-h-z — COMMENTED** [review]
Looks good overall!

Please take a look at any code comments, if any.

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -8,19 +8,7 @@
...
-import seedu.address.logic.commands.Command;
-import seedu.address.logic.commands.DeleteAppointmentCommand;
-import seedu.address.logic.commands.DeleteClientProfileCommand;
-import seedu.address.logic.commands.DeletePropertyCommand;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.ExitCommand;
-import seedu.address.logic.commands.FindCommand;
-import seedu.address.logic.commands.HelpCommand;
-import seedu.address.logic.commands.ListCommand;
+import seedu.address.logic.commands.*;
```
Comment (lrongyi) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:11]: Avoid wildcard imports

File: src/main/java/seedu/address/model/AddressBook.java
Comment (lrongyi) [inline src/main/java/seedu/address/model/AddressBook.java:1]: LGTM

**Review by lrongyi — COMMENTED** [review]
Overall LGTM, just require testing and integration with the Listing Model itself

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 2 above)
Comment (muhdbhz) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java]: Resolved

File: src/main/java/seedu/address/model/AddressBook.java (hunk 1 above)
Comment (muhdbhz) [reply src/main/java/seedu/address/model/AddressBook.java:104]: Resolved

**Pull request closed without merging**